pages/table/expandable-rows-test.page.tsx

```diff
@@ -355,6 +370,7 @@ function usePageSettings() {
     useProgressiveLoading: urlParams.useProgressiveLoading ?? true,
     groupResources: urlParams.groupResources ?? true,
     useServerMock: urlParams.useServerMock ?? false,
+    manualServerMock: urlParams.manualServerMock ?? false,
```

Comment:
What's the difference between this and using the above `useServerMock`?

Reply:
Use server mock activates async behaviour with some default 1.5s delay which is helpful for manual testing. The manualMock replaces the delay with a flush command.